Makefile

```diff
@@ -1,3 +1,5 @@
+TEST_DIR := docs/source/examples/petstore
```

Comment:
this is not the only test dir - that was reason to not have it here

Reply:
if you wanted to generalize this, you would need to make this TEST_DIRS - list of dirs